luxonis_ml/data/augmentations/utils.py

```diff
@@ -287,7 +373,8 @@ def post_transform_process(
         ns: int,
         nk: int,
         filter_kpts_by_bbox: bool,
-    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+        return_mask: bool = True,
+    ) -> Tuple[np.ndarray, Optional[np.ndarray], np.ndarray, np.ndarray]:
```

Comment:
From type complience perspective should we make default `out_mask=np.zeros(...)` instead of `None` since `__call__` should return `Dict[LabelType, np.ndarray]`? cc: @kozlov721

Reply:
We don't return mask if it's `None`.

Reply:
Where don't we return it? Because we return it from `post_transform_process` as `None` and then this is part of out_annotations in `__call__`'s return right?

Reply:
If mask is in `present_annotations` we return it. Otherwise we return None from `post_transform_process` and nothing from `__call__`. CC: @JSabadin to confirm